src/puptoo/process/profile.py

```diff
@@ -44,6 +45,20 @@ def catch_error(parser, error):
     logger.error(log_msg, parser, error)
 
 
+# GCP_CONFIRMED_CODES are the available marketplace license codes available
+# from the Google Compute Platform. These may need to be updated regularly.
```

Comment:
@CodeHeeler I've thought about this quite a bit and I think it's certainly worth swinging around to it. The issue I have is that I certainly don't want to check for the proper license keys every time we get a payload with GCP. That could lock us up since this is all synchronous. The other thing is that they hide the license keys behind auth in their API, so we'd have to get our tokens all set and then deal with them if they expire.

For now, i think we should just roll with it being not automated, but maybe we can devise some sort of notification either in buidls or PR checks that lets us know we have new licenses to check for. I'm sure there's a lot we can do here.

Reply:
Yeah, that makes sense.  Maybe just create a jira or something so we can revisit it.